src/clib/lib/job_queue/job_node.cpp

```diff
@@ -209,8 +167,7 @@ ERT_CLIB_SUBMODULE("queue", m) {
         std::optional<std::string> error_msg = std::nullopt;
 
         if (current_status & JOB_QUEUE_RUNNING && !node->confirmed_running) {
-            node->confirmed_running =
-                !node->status_file || fs::exists(node->status_file);
+            node->confirmed_running = !fs::exists(node->status_file);
```

Comment:
Is this equivalent with the replaced code?
It used to be true if string was empty OR the file actually existed?

Reply:
It was for the possibility of setting status_file to nullptr, which we don't do.

Reply:
Oh shit, I guess we need to fix that. We don't really use that method any more, but we might want to investigate what is going on.